src/_buttons.scss

```diff
@@ -0,0 +1,64 @@
+@use 'sass:color';
+
+//Cretae a map of colors
+$colors: (
+  "primary": $fe-clr-primary,
+  "secondary": $fe-clr-secondary,  //accordign to the draft it should be purple but that color code is not in the globalvariables.
+  "success": $fe-clr-success,
+  "info": $fe-clr-info,
+  "warning": $fe-clr-warning,
+  "danger": $fe-clr-danger,
+  "default": $fe-clr-dark,
+);
+
+//Create a mixin for buttons
+@mixin btn() {
```

Comment:
I suggest moving this mixin to `_mixins.scss`

Reply:
OK, thank you. 
I moved the mixins code from the _buttons to _mixins file. 
Already pushed the changes.